### DISCUSSION TITLE
Does the Crowd relay work work as a proxy?

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1683] [Developer Support]
"Hi. I saw you updated the documentation about the Cloud Relay (https://docs.walletconnect.com/2.0/cloud/relay). Is my understanding correct, that I can register a Relay Project ID and get an URI (e.g. https://relay.walletconnect.com/?projectId=c4f79cc821944d9680842e34466bfbd) that I can use to whitelist all incoming and outgoing traffic? Does it work like a proxy"

**Comment by arein** [comment]
One of the features a projectid gives you is the ability to allowlist traffic from certain domains (i.e. prevent other websites to use your project id).
It also allows us to link your dapp/wallet to our Cloud App where your dapp/wallet's settings are stored e.g. the webhook url for push notifications.
Lastly, we collect some anonymized metrics such as the country of where the users are calling us from and we will soon expose those metrics in your project's page in Cloud App.
